src/basic/Root.js

```diff
@@ -14,12 +14,12 @@ class Root extends Component {
         {this.props.children}
         <Toast
           ref={c => {
-            if (!Toast.toastInstance) Toast.toastInstance = c;
+            if (c) Toast.toastInstance = c;
```

Comment:
I think you don't need to check if `c` is different of null.

Reply:
Thanks for taking a look at this @douglasjunior. Does GeekyAnts have a preference on how to handle the case where [`c` is `null`](https://reactjs.org/docs/refs-and-the-dom.html)? @akhil-geekyants @SupriyaKalghatgi

Reply:
https://reactjs.org/docs/refs-and-the-dom.html#callback-refs

> React will call the ref callback with the DOM element when the component mounts, and call it with null when it unmounts. ref callbacks are invoked before componentDidMount or componentDidUpdate lifecycle hooks.

I think if the component has been unmounted, you should not keep the ref. This is memory leak.